src/ontology/oba.Makefile

```diff
@@ -139,14 +160,24 @@ $(REPORTDIR)/%.tsv: ../sparql/synonyms-exact.sparql $(TMPDIR)/%.owl
 prepare_oba_alignment: $(REPORTDIR)/uberon.tsv $(REPORTDIR)/efo.tsv $(REPORTDIR)/pato.tsv $(REPORTDIR)/oba.tsv $(REPORTDIR)/vt.tsv $(REPORTDIR)/cl.tsv $(REPORTDIR)/go.tsv $(REPORTDIR)/chebi.tsv
 	echo "OK cool all tables prepared."
 
-CHECK_SPARQL=oba.owl
+#This is the OBA-VT DIFF Pipeline
+
+$(TMPDIR)/%.owl:
+	wget $(OBOBASE)/$*.owl -O $@
+
+$(TMPDIR)/simple-%.owl: $(TMPDIR)/%.owl data/vtoba_terms.txt 
+	$(ROBOT) merge -i $< \
+	filter -T data/vtoba_terms.txt  --select 'self annotations children' --signature true -o $@
```

Comment:
```suggestion
	remove -T data/vtoba_terms.txt --select complement --preserve-structure false --trim false -o $@
```

Reply:
@souzadevinicius try this and see if the hierarchies look a bit better (and run the diff again). If you have oak installed locally, upgrade to latest version and try the new bidirectional parameter. Check PRs/Docs how to use.

Reply:
As a result of this change, we see more hierarchies in protege, but in the diff file itself, the categories are unbalanced and probably still different from what we want to see.

## VT_SIMPLE 
![VT_SIMPLE](https://user-images.githubusercontent.com/6750638/207395113-9723f025-0cb9-4a17-a86e-b9d1175e6011.png)

## OBA_SIMPLE

![image](https://user-images.githubusercontent.com/6750638/207395560-e906fd0a-8466-468e-9f80-35197559aa2d.png)


## Diff File Categories

| name                     | total |
|--------------------------|-------|
| category: Identical      |     1 |
| category: MissingMapping |  8449 |

Reply:
Before you do anything else: You have not committed my suggestion it seems? The simple files should look perfect before attempting to get anything meaningful out of them. 

You can look at OLS to get a sense how they should look like: 

https://www.ebi.ac.uk/ols/ontologies/oba/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBA_0000001